src/extra/gfx/feature.cpp

```diff
@@ -15,27 +15,37 @@
 
 using namespace shards;
 
+namespace gfx {
+enum class BuiltinFeatureId { Transform, BaseColor, VertexColorFromNormal, Wireframe, Velocity };
+}
+
+ENUM_HELP(gfx::BuiltinFeatureId, gfx::BuiltinFeatureId::Transform, SHCCSTR("Add basic world/view/projection transform"));
+ENUM_HELP(gfx::BuiltinFeatureId, gfx::BuiltinFeatureId::BaseColor,
+          SHCCSTR("Add basic color from vertex color and (optional) color texture"));
+ENUM_HELP(gfx::BuiltinFeatureId, gfx::BuiltinFeatureId::VertexColorFromNormal, SHCCSTR("Outputs color from vertex color"));
+ENUM_HELP(gfx::BuiltinFeatureId, gfx::BuiltinFeatureId::Wireframe, SHCCSTR("Modifies the main color to visualize vertex edges"));
+ENUM_HELP(gfx::BuiltinFeatureId, gfx::BuiltinFeatureId::Velocity,
+          SHCCSTR("Outputs object velocity into the velocity global & output"));
```

Comment:
@ChelseaChX Should we have a final `.` for those help strings? I added one on my own enum description since we also had those on the regular `help()` function.

Reply:
Yeah I think we have them everywhere else in help strings, can add them here.

Reply:
> @ChelseaChX Should we have a final `.` for those help strings? I added one on my own enum description since we also had those on the regular `help()` function.

Yeah we should keep it consistent.